src/main/java/org/AuthenticateDbHandler.java

```diff
@@ -196,6 +196,56 @@ private String getSecretKey() {
     	return null;
 	}
 
+	public Claims getClaims(WebTokens compactJws) {
```

Comment:
Another method, `public int getInt(String key, WebTokens compactJws)` is almost the same as `public Claims getClaims(WebTokens compactJws)`